tests/test_fastapi_dx.py

```
@@ -0,0 +1,22 @@
+        authentication_mode=AuthenticationMode.DISABLED,
+        runtime_profile=RuntimeProfile.DEVELOPMENT,
+    )
+    app = create_app(settings)
+    assert app.swagger_ui_parameters.get("persistAuthorization") is True
+
+def test_production_profile_does_not_persist_authorization():
+    """Verify persistAuthorization is not enabled in production profile."""
+    settings = RuntimeSettings(
+        authentication_mode=AuthenticationMode.REQUIRED,
+        runtime_profile=RuntimeProfile.PRODUCTION,
+        api_token="test_token"
```

Ruff (0.16.3)

[error] Possible hardcoded password assigned to argument: "api_token"

(S106)